src/Codeception/Module.php

```diff
@@ -345,7 +345,7 @@ protected function getModule($name)
     /**
      * Get config values or specific config item.
      *
-     * @param null $key
+     * @param mixed|null $key
```

Comment:
Is there any point to keep |null after mixed?
I think that this line and return line can be simplified to mixed type.
`@return array|mixed|null` -> `@return mixed`

Reply:
1/ maybe not
2/ agreed